### ISSUE TITLE
Ruby 2.7 deprecations

### ISSUE THREAD

**Issue by nickpellant** [rom-rb/rom-factory#57]
**Describe the bug**

`rom-factory` causes deprecation warnings when used with Ruby 2.7.

**To Reproduce**

* Use `rom-factory` with Ruby 2.7
* See warnings in logs for RSpec

**Example**

> ruby/2.7.0/lib/ruby/gems/2.7.0/gems/rom-factory-0.10.1/lib/rom/factory/dsl.rb:18: warning: Using the last argument as keyword parameters is deprecated; maybe ** should be added to the call

**Expected behavior**

It should not cause deprecation warnings.

**Your environment**

- Ruby version: 2.7.0

**Comment by flash-gordon** [comment]
@nickpellant please provide code that emits a warning

**Comment by nickpellant** [comment]
@flash-gordon In my codebase, this has to do with the usage of calls to `fake` in factories, i.e:

```ruby
Factory.define(:entity) do |f|
  f.entity_number { fake(:number, :within, range: -20..20) }
end
```

**Comment by flash-gordon** [comment]
Thanks for reporting, fixed in 0.10.2
